akka-testkit/src/main/scala/akka/testkit/javadsl/TestKit.scala

```diff
@@ -732,8 +733,18 @@ class TestKit(system: ActorSystem) {
    *
    * This method does NOT automatically scale its Duration parameter!
    */
+  @Deprecated
+  @deprecated("Use the overloaded one which accepts java.time.Duration instead.", since = "2.5.13")
   def receiveOne(max: Duration): AnyRef = tp.receiveOne(max)
 
+  /**
+   * Receive one message from the internal queue of the TestActor. If the given
+   * duration is zero, the queue is polled (non-blocking).
+   *
+   * This method does NOT automatically scale its Duration parameter!
```

Comment:
Should we take this opportunity to also dilate this parameter, or do we prefer not to by design?

Reply:
Do you mean method `dilated`?  `receiveOne` just call scala version of `receiveOne`:
https://github.com/akka/akka/blob/de97fbe8767f864d1d15afd8d7729433cda765ae/akka-testkit/src/main/scala/akka/testkit/TestKit.scala#L771-L795

If this what you told about, I think we could file another issue to discuss. Since this ticket is just adding overloaded method.

Reply:
Ah, having it inconsistent between Java and Scala would be weird. No need for a separate ticket then, this is :ok_hand: